### PR TITLE
 Wrote a site to explain the object headers

### PR DESCRIPTION
**Pull request by Mr-Brownstone**

### CODE REVIEW

**Comment by TimNN** [comment]
I've pushed a commit fixing some of the metadata, this was easier than commenting on everything.

File: notes/content/02_imgseg/object-header.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+| 1             | fixed pointer fields                                |
+| 2             | indexable pointer fields                            |
+| 3             | both fixed and indexable pointer fields             |
+| 4+5           | not used                                            |
+| 6             | indexable word fields (no pointer)                  |
+| 7             | not used                                            |
+| 8-11          | indexable byte fields (no pointers): low 2 bits are low 2 bits of size in bytes |
+| 12-15         | compiled methods: low 2 bits are low 2 bits of size in bytes. The number of literals is specified in method header, followed by the indexable bytes that store byte codes. |
+
+The size in in words stores the objects size if it isn't bigger than 64 words.
```
Comment (TimNN) [inline notes/content/02_imgseg/object-header.md]: Something is wrong with this sentence ("The size in in words"), but I don't known what it is supposed to say, so I can't fix it.

**Comment by TimNN** [comment]
@Mr-Brownstone: Looks good so far and I believe this can be merged as soon as the last sentence (see comment) is fixed. Please remember to pull before making any changes as to not override the commits I added to this PR.

I'd also like @Felerius or @Metauriel to review this as well, since they are more familiar with the topic.

**Comment by Felerius** [comment]
@Mr-Brownstone All the information looks correct. Looks good to merge once the one sentence @TimNN remarked on has been changed (to something like "the `size in words` field is used to store the object size unless it exceeds 64 words").

**Pull request merged**